src/DeploymentManager/DeploymentManager/Commands/SetServiceTopology.cs

```diff
@@ -0,0 +1,46 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+namespace Microsoft.Azure.Commands.DeploymentManager.Commands
+{
+    using System.Management.Automation;
+
+    using Microsoft.Azure.Commands.DeploymentManager.Models;
+
+    [Cmdlet(
+        VerbsCommon.Set, 
+        ResourceManager.Common.AzureRMConstants.AzurePrefix + "DeploymentManagerServiceTopology",
+        SupportsShouldProcess = true,
+        DefaultParameterSetName = DeploymentManagerBaseCmdlet.InputObjectParamSetName),
+     OutputType(typeof(PSServiceTopologyResource))]
+    public class SetServiceTopology : DeploymentManagerBaseCmdlet
```

Comment:
@deveshguha same comment about the two new sets

Reply:
This is not an update to our cmdlets, but merely a move from AzureRM to Az format, so do not want to add more funtionality at this time, for this preview version. Will update these in the subsequent versions.